Project-Uni/RootShare
# Team info

**Updates**
1. Adds team info to homepage
2. Adds loading indicator to admin count page

**Testing**
- View the homepage, see that team info looks good on desktop and mobile
- View admin count page. See that new loading indicator works fine.
